bot/exts/moderation/stream.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import logging
+
+import discord
+from async_rediscache import RedisCache
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Emojis, Guild, Roles, STAFF_ROLES, VideoPermission
+from bot.converters import Expiry
+from bot.utils.scheduling import Scheduler
+from bot.utils.time import format_infraction_with_duration
+
+log = logging.getLogger(__name__)
+
+
+class Stream(commands.Cog):
+    """Grant and revoke streaming permissions from users."""
+
+    # Stores tasks to remove streaming permission
+    # User id : timestamp relation
+    task_cache = RedisCache()
```

Comment:
Uhh, the variable name is possibly giving the wrong information, it doesn't store "task" cache, but rather the user ids. for which it has to remove the video roles.

Reply:
It stores a list of userIDs that need to be scheduled to be removed, I think I agree with the original author that `task_cache` is the best way to describe this.

Reply:
Well I can't come up with something concise and better, so lets go with this.